supabase/functions/tests/.env.edge_testing

```
@@ -7,3 +7,5 @@ IS_TESTING=true
 SHARED_CSV_CAMPAIGN_LABEL_ID=abc
 DUB_API_KEY=abc
 DUB_TAG_NAME=txt-messages
+OUTLIER_PHONE_NUMBER=+15555550100
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OUTLIER_PHONE_NUMBER key should go before the SHARED_CSV_CAMPAIGN_LABEL_ID key

(UnorderedKey)